src/Notifications.svelte

```diff
@@ -168,51 +171,37 @@
   }
 
   export let timeout = 3000
+  export let closeable = true
 
 	let count = 0
-	let toasts = [ ]
+	let toasts = []
   let unsubscribe
 
-	function animateOut(node, { delay = 0, duration = 1000 }) {
-		function vhTOpx (value) {
-			var w = window,
-				d = document,
-				e = d.documentElement,
-				g = d.getElementsByTagName('body')[0],
-				x = w.innerWidth || e.clientWidth || g.clientWidth,
-				y = w.innerHeight|| e.clientHeight|| g.clientHeight;
-
-			return (y*value)/100;
-		}
-		
-		return {
-			delay,
-			duration,
-			css: t => `opacity: ${(t-.7) * 1}; transform-origin: top right;`
-		}
-	}
+	const animateOut = (node, { delay = 0, duration = 1000 }) => ({
```

Comment:
weird, I had no idea we didn't use this. nice touch.

Reply:
VSCode syntax highlighting really comes in handy these moments 😁